themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,34 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group']): ?>
+                    <?php if (isset($this->config->AuthorResultListInfo->ResultListPage) && ($this->config->AuthorResultListInfo->ResultListPage == 'Short')): ?>
+                        <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+			<?php $authorCount = count($authors) ?>
+		        <?php $authorNameLimit = $this->config->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                        <?php if (isset($authors) && !empty($authors)): ?>
+                            <b><?=$this->transEsc($item['Label'])?>:</b>
+                            <?php $count = 0; ?>
+                            <?php foreach ($authors as $author): ?>
```

Comment:
Could you use `foreach ($authors as $i => $author)` here and then use `($i + 1)` in place of `$count` to eliminate the need for a separate counter? Or is the author array indexing unreliable/unpredictable?

Reply:
Thanks @demiankatz . I don't beleive author array indexing is unreliable so we should be able to use this. I've added it in my follow-up